packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -600,14 +600,13 @@ class ChartPainter extends CustomPainter {
   }
 
   TextPainter createText(String textValue, double scale) {
-    const TextStyle(
```

Comment:
@pq why don't our dead code lints catch const values that can never be referenced like this one? If this was a non-const class maybe it could have side effects but as is it is pointless.

Reply:
This is really interesting. Normal constructors can be side effecting so we can never really treat them as dead but const constructors, I think you're right could be flagged.

Reply:
Where should a bug to track this be filed?